sdk/compute/arm-compute/package.json

```diff
@@ -49,9 +50,16 @@
     "tsconfig.json"
   ],
   "scripts": {
-    "build": "tsc && rollup -c rollup.config.js && npm run minify",
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
```

Comment:
Make sure that audit key in scripts is not generated in package.json with relative reference paths in vanilla flavor of autorest.typescript. This is very much specific to the way things are structured in azure-sdk-for-js.

It would be nice if we create small packages for such scripts and then use those packages. In that way everyone can benefit from it.

Reply:
There doesn't seem to be a mechanism for Azure-flavored package.json mods so I'll have to come up with a way to do that.  Is there anything else you see here that shouldn't be included in the vanilla-flavored output?

Reply:
No longer adding `audit` script, see this comment: https://github.com/Azure/azure-sdk-for-js/pull/4237#issuecomment-510600408